project-base/config/packages/snc_redis.yaml

```diff
@@ -6,24 +6,12 @@ snc_redis:
             dsn: 'redis://%redis_host%'
             options:
                 prefix: '%env(REDIS_PREFIX)%%build-version%:cache:bestselling_products:'
-        doctrine_metadata:
-            type: 'phpredis'
-            alias: 'doctrine_metadata'
-            dsn: 'redis://%redis_host%'
-            options:
-                prefix: '%env(REDIS_PREFIX)%%build-version%:cache:doctrine:metadata:'
         doctrine_query:
             type: 'phpredis'
             alias: 'doctrine_query'
             dsn: 'redis://%redis_host%'
             options:
                 prefix: '%env(REDIS_PREFIX)%%build-version%:cache:doctrine:query:'
```

Comment:
Doctrine query is not problematic? Why did it stay here?

Reply:
The most problematic is `framework annotations` because lot of entries into cache on page load. I also moved `metadata cache` because it's generated once and they are not changing in running application. Both 

Doctrine queries are dynamically generated, so I prefer to have them in Redis cache because write to Redis is faster. 

Look at screen. There is count of queries grouped by cache type on stabil Homepage
<img width="975" alt="Snímek obrazovky 2020-11-18 v 12 41 22" src="https://user-images.githubusercontent.com/1482966/99526771-088c9500-299c-11eb-9652-d2dff7fe0446.png">

Reply:
Ok, thank you for explanation